docs/contributing/contributing-code/contributing-code-control-plane/running-controlplane-locally.md

```diff
@@ -26,12 +22,13 @@ If you need to manually test APIs you can reach them at the following endpoints
 
 ## Prerequisites
 
-1. Create a Kubernetes cluster, or set your current context to a cluster you want to use. The debug configuration will use your current cluster for storing data.
-2. Clone the `radius-project/radius` and `radius-project/deployment-engine` repo next to each other.
-3. Run `git submodule update --init` in the `deployment-engine` repo.
-4. Install .NET 8.0 SDK - <https://dotnet.microsoft.com/en-us/download/dotnet/8.0>.
-5. Install C# VS Code extension - <https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp>.
-6. (Optional) Configure any cloud provider credentials you want to use for developing Radius.
+1. Create a Kubernetes cluster, or set your current context to a cluster you want to use. The debug configuration will use your current cluster for storing data. 
+2. If you have access to `radius-project/deployment-engine` repo, 
+   1. Clone the `radius-project/radius` and `radius-project/deployment-engine` repos next to each other.
```

Comment:
Same comment as before https://github.com/radius-project/radius/pull/7964#discussion_r1929940676, I might not have worded it clearly earlier. Why is cloning the repo still a prerequisite?